src/ooxml/java/org/apache/poi/ooxml/POIXMLProperties.java

```diff
@@ -376,110 +376,164 @@ public String getTemplate() {
             }
             return null;
         }
+        public void setTemplate(String template) {
```

Comment:
could you add `@since` javadoc annotations on the new methods - eg `@since POI 4.1.1`?

No need to much javadoc besides. Code is pretty self-explanatory.